credstash.py

```diff
@@ -393,6 +393,12 @@ def main():
                                  help="Put a specific version of the "
                                  "credential (update the credential; "
                                  "defaults to version `1`).")
+    parsers[action].add_argument("-a", "--autoversion", action="store_true",
+                                help="Automatically increment the version of "
+                                "the credential to be stored. This option "
+                                "causes the `-v` flag to be ignored. "
+                                "(This option assumes that the version "
+                                "currently stored is numeric.)")
```

Comment:
Might be clearer with some variation of "operation will fail if the current version is not numeric"